charts/rancher-turtles/Chart.yaml

```diff
@@ -13,7 +13,7 @@ keywords:
 - provisioning
 dependencies:
   - name: cluster-api-operator
-    version: v0.5.1
+    version: v0.7.0
```

Comment:
can you also bump all operator manifests to `v1alpha2`? Can be done in a follow-up PR

Reply:
I think we can do this in a followup PR. CAPI operator uses v1alpha2 as a store version, so submitting v1alpha1 is still stored as v1alpha2 and is not critical.

Reply:
Unfortunately there is a bug in v0.7.0 preventing us from using it. Reverted it to 0.6.0, as this one works in testing.

Reply:
Should be eventually fixed by https://github.com/kubernetes-sigs/cluster-api-operator/pull/352.